src/data-table/DataTableContextMenu.component.js

```diff
@@ -22,9 +22,10 @@ function DataTableContextMenu(props, context) {
     const cmStyle = {
         position: 'fixed',
     };
+    const popoverProps = _.pick(props, _.keys(Popover.propTypes));
```

Comment:
The original code passes all the DataTableContextMenu props to the Popover. Of course, most of those props don't really exist, so a warning is displayed. `_.pick` filters the props by keeping only the defined in Popover.